redbiom/admin.py

```diff
@@ -209,7 +256,91 @@ def create_context(name, description):
     ScriptManager.load_scripts()
 
 
-def load_sample_data(table, context, tag=None, redis_protocol=False):
+def _load_axis_data(table, ids, opposite_ids, opposite_id_index, axis_label,
+                    context, batchsize):
+    """Manage the loading of data for a particular axis
+
+    Parameters
+    ----------
+    table : biom.Table
+        The table to obtain data from
+    ids : iterable of str
+        The IDs to obtain data for
+    opposite_ids : iterable of str
+        The IDs of the opposite axis in the table
+    opposite_id_index : dict
+        The index which maps an opposite ID to the index value within
+        the Redis database for the identifier
+    axis_label : str
+        The biom.Table axis label of ids
+    context : str
+        The context to load the data into
+    batchsize : int
+        The number of identifiers to group into a single request
+
+    Notes
+    -----
+    This method does not support non count data.
+
+    Data are loaded through the "load-data" Lua script managed in the
+    ScriptsManager. This method in effect packs the data into a structure
+    compatible with Webdis, and the EVALSHA command structure of Redis. The
+    "load-data" script then iterates over the "KEYS" and "ARGV"s, parsing
+    the respective entries into values that can be directly loaded.
+
+    Redis command summary
+    ---------------------
+    EVALSHA <load-data-sha1> N <context>:<axis_label>:<id> ... <packeddata> ...
+
+    Note that "N" refers to the number of "KEYS". The "load-data" Lua script
+    assumes that there are "N" "KEYS" as well as "N" "ARGV"s. For the call,
+    "KEYS" are the prefixed identifiers (e.g., "<context>:<axis_label>:<id>")
+    and "ARGV" are the "packeddata". "KEYS" and "ARGV" are expected to be in
+    index order with each other.
+    """
+    import redbiom
+    import redbiom._requests
+    if axis_label == 'feature':
+        axis = 'observation'
+    else:
```

Comment:
Just for sanity, should this be an elif and raise an error if not feature or sample? Perhaps, this will never happen?!